server/views/components/series-container/index.njk

```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="container">
-    <div class="grid">
+    <div class="grid grid--dividers">
```

Comment:
The `.grid--dividers` class seems like a pretty reusable thing, why not turn it into a mixin and use different classes to denote parts of the grid that need the dividers? Something like `grid--articles`, which could then have additional styling independent of the dividers.